optimism/src/keccak/column.rs

```diff
@@ -49,12 +48,10 @@ pub enum KeccakColumn {
     PiRhoExpandRotE(usize, usize, usize),     // Round Curr[1065..1165)
     ChiShiftsB(usize, usize, usize, usize),   // Round Curr[1165..1565)
     ChiShiftsSum(usize, usize, usize, usize), // Round Curr[1565..1965)
-    IotaStateG(usize),                        // Round Next[0..100)
-    SpongeOldState(usize),                    // Sponge Curr[0..100)
     SpongeNewState(usize),                    // Sponge Curr[100..200)
     SpongeBytes(usize),                       // Sponge Curr[200..400)
     SpongeShifts(usize),                      // Sponge Curr[400..800)
-    SpongeXorState(usize),                    // Absorb Next[0..100)
+    Output(usize),                            // Next[0..100) either IotaStateG or SpongeXorState
```

Comment:
Note that in [witness](https://github.com/o1-labs/proof-systems/blob/dfd5056d46a5960f8933d5979d4dcb6aaa11afc8/optimism/src/keccak/witness.rs#L147) we could also rename.